x-pack/plugins/stack_connectors/server/connector_types/auth/schema.ts

```diff
@@ -33,16 +33,16 @@ export const AuthConfiguration = {
   ),
 };
 
-export const SecretConfiguration = {
+export const SecretConfiguration = schema.object({
   user: schema.nullable(schema.string()),
   password: schema.nullable(schema.string()),
   crt: schema.nullable(schema.string()),
   key: schema.nullable(schema.string()),
   pfx: schema.nullable(schema.string()),
-};
+});
 
 export const SecretConfigurationSchemaValidation = {
-  validate: (secrets: any) => {
```

Comment:
`any` is flagged as an error in the `server` but not in `common` 😅 

So I chose to remove it instead of skipping it.